@kiva/kv-components/utils/loanUtils.js

```diff
@@ -56,33 +71,36 @@ export function getLendCtaSelectedOption(
 			// eslint-disable-next-line no-nested-ternary
 			currentCampaign = campaignToCheck.includes(TOP_UP_CAMPAIGN)
 				? TOP_UP_CAMPAIGN
-				: (campaignToCheck.includes(BASE_CAMPAIGN) ? BASE_CAMPAIGN : '');
+				: campaignToCheck.includes(BASE_CAMPAIGN) ? BASE_CAMPAIGN // eslint-disable-line no-nested-ternary
+					: campaignToCheck.includes(BALANCE_CAMPAIGN) ? BALANCE_CAMPAIGN // eslint-disable-line no-nested-ternary
+						: (campaignToCheck.includes(NO_BALANCE_CAMPAIGN) ? NO_BALANCE_CAMPAIGN : '');
```

Comment:
This is getting pretty messy. We'll likely want to find a better way if these campaign types keep expanding...

Reply:
Agreed!
Only issue I see is that we have different conditions for every campaign. Could we get them from kiva admin?